src/packages/core/src/metadata-service/resolver.ts

```diff
@@ -137,8 +137,12 @@ export const resolveAdminUiMetadata = (hooks?: Hooks) => {
 					fieldObject.relationshipType = RelationshipType.ONE_TO_MANY;
 
 					const relatedEntityField = Object.values(relatedObject.fields).find((field) => {
-						const fieldType = field.getType() as { name?: string };
-						return fieldType.name === (entity.target as { name?: string }).name;
+						const { isList: isOtherSideList, fieldType: fieldTypeOtherSide } =
```

Comment:
I couldn't get relationship fields to be detected at this point. This should be a more accurate matching check